### ISSUE TITLE
Make new Behavior Instances send indicators

### ISSUE THREAD

**Issue by tobydragon** [tobydragon/wizard#2]
The behavior instances are apparently created, but not sent yet. Probably in the RunestoneInterventionController

**Comment by tobydragon** [comment]
Since we don't have an xmpp server, we should have a simpler communication scheme, new issue.

**Comment by tobydragon** [comment]
watiing on #16.

**Comment by tobydragon** [comment]
blocked on #16

**Comment by tobydragon** [comment]
fixed, along with #16
